lib/Toolbar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import icons from './icons';
+
+class Toolbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { showHeaders: false };
+  }
+
+  render() {
+    return (<div id="quill-toolbar">
```

Comment:
`id` should use a prop.

Reply:
Just a minor piece of feedback to but check linting as I would have thought that the div would need to be on a new line?